README.rst

```diff
@@ -39,12 +39,15 @@ with frameworks even better.
 -   `WTForms-Django`_ provides ORM-backed fields and form generation
     from Django models, as well as integration with Django's I18N
     support.
+-   `Starlette-WTF`_ integrates with Starlette/FastAPI framework.It is 
```

Comment:
The style failure comes from the space at the end of this line. I think it should pass if you remove the space.